src/pages/General/Profile.tsx

```diff
@@ -0,0 +1,122 @@
+import axios from "axios";
+import Auth from "../../components/utils/Auth";
+import BACKEND_URL from "../../components/utils/Constants";
+import def_Img from "../../images/default.png";
+import CardSlider from "../../components/CardSlider";
+import "../../styles/profile.css";
+import { useState, useEffect} from "react";
+import editIcon  from "../../images/edit_icon.svg"
+import { makeAuthenticatedPostRequest } from "../../components/utils/Helpers";
+const DataField = (props:any) => {
+
+    return(
+        <div className="dataField">
+            <input className ="profileEditInput" type="text" id={props.name} defaultValue={props.value} disabled/>    
+        </div>
+    );
+}
+const Profile = (user : any) => {
+    const[details, setDetails] =  useState<any[]>([]);
+    useEffect(()=>{
+        let headers = {
+            "Authorization": `Token ${Auth.getToken()}`
+        }
+        axios.get(`${BACKEND_URL}/get_user_details/?username=${Auth.getUsername()}`, {headers: headers})
+        .then(response => {
+            setDetails(response.data);  console.log(response.data);
+            console.log(details.length);          
+        })
+        .catch((error) => {
+            // TODO: actually handle this error
+            console.log(error.response.data);
+            console.log(error.request);
+            console.log(error.message);
+        });
+    },[]);
+
+    const editData = () =>{
+        // Swap button
+        document.getElementById("editProfileBtn")!.style.display="none";
+        document.getElementById("submitChangesBtn")!.style.display="inline";
+
+        // Enable input fields        
+        const fields = document.getElementsByClassName("profileEditInput");
+        Array.prototype.filter.call(fields, function(field){
+            field.disabled = false;
+            console.log(field.value);
+        });
+        
+    }
+    const submitChanges = () =>{
+        // get input data
+        const username : any = document.getElementById("username");
+        const email : any = document.getElementById("email");
+        console.log(username.value, email.value);
+   
+        let body = {
+            "username": username.value,
+            "current_username": Auth.getUsername(),
+            "email": email.value,      
+            "subscriptions" : details[0].subscriptions,
+          }
+
+        // update request
+         makeAuthenticatedPostRequest(
+            "/update_user_details/", 
+            "Success! User details updated!", 
+            body);
+
+        // update local storage
+        localStorage.setItem("username", JSON.stringify(username.value));
+    }
+    
+    return(
+        <div>
+            {(details.length==1)?( <>
+                <main>
+                <h3>Profile</h3> 
+                <form onSubmit={submitChanges}>         
+                    <div className ="profile-wrapper">  
+                    <img 
+                        src={details[0].image}
```

Comment:
Change this line to: src={`${BACKEND_URL}/media/${details[0].image}`}
This will actually fetch the correct image from the backend.